test/test-default-logger.rb

```diff
@@ -52,8 +54,9 @@ def test_plus
     end
 
     def test_string
-      file = Tempfile.new("chupa-text-default-logger-output")
-      assert_equal(file.path, output(file.path).path)
+      Tempfile.create("chupa-text-default-logger-output") do |file|
+        assert_equal(file.path, output(file.path).path)
```

Comment:
`output`の変更はなしにしてこっちで`close`かな。

```suggestion
        device = output(file.path)
        device.close
        assert_equal(file.path, device.path)
```